src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import NavBar from './components/Navbar';
+import BookList from './components/BookList';
+import Categories from './components/Categories';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<BookList />} />
+        <Route path="/categories" element={<Categories />} />
```

Comment:
- [ ] While you make a great effort 😃I see that you created the ```<Categories>``` component like the requirements, but I see that you didn't create ```<books>``` component which contains ```<booklist>``` component I see that you put the ```<booklist>``` without  ```<books>``` parent component, so please try to create the ```<books>``` component and up the ```<booklist>``` component inside it to follow the requirements 💯🚀

![image](https://github.com/Benawi/Microverse-React-Bookstore/assets/83360636/5ba1200e-194a-401d-9f72-e7bf292ccabf)